### ISSUE TITLE
Can't compile Flurry module with Godot 2.0

### ISSUE THREAD

**Issue by kubecz3k** [punto-/godot_modules#2]
Since couple days (with some pauses) I'm trying to make flurry module work with Godot 2.0
Basing on http://docs.godotengine.org/en/latest/reference/creating_android_modules.html I made couple changes
- in `GodotFlurry.java` changed package to `org.godotengine.godot` 
- Corrected localisation of GodotFlurry.java to meet the package (`src/org/godotengine/godotGodotFlurry.java`)
- changed config.py to meet 2.0 style (tried couple variations, below is android section of my config.py also with different tries commented)

```
    elif env['platform'] == 'android':
        env.android_add_java_dir("android")
        env.android_add_dependency("compile files('android/FlurryAgent.jar')") 
        env.disable_module()

#       env.android_add_java_dir("android/src/org/godotengine/godot")
#       env.android_add_dependency("compile 'com.flurry.android:analytics:6.3.1'")
#       env.android_module_file("android/GodotFlurry.java") #before 2.0
```

Running ./gradlew build always fails because java can't recognize FlurryAgent class

```
kubeczek@kubeczek-K52JT:~/git/godotBase/platform/android/java$ ./gradlew build
:preBuild UP-TO-DATE
:preDebugBuild UP-TO-DATE
:checkDebugManifest
:prepareDebugDependencies
:compileDebugAidl UP-TO-DATE
:compileDebugRenderscript UP-TO-DATE
:generateDebugBuildConfig UP-TO-DATE
:generateDebugAssets UP-TO-DATE
:mergeDebugAssets UP-TO-DATE
:generateDebugResValues UP-TO-DATE
:generateDebugResources UP-TO-DATE
:mergeDebugResources UP-TO-DATE
:processDebugManifest UP-TO-DATE
:processDebugResources UP-TO-DATE
:generateDebugSources UP-TO-DATE
:processDebugJavaRes UP-TO-DATE
:compileDebugJava
/home/kubeczek/git/godotBase/modules/flurry/android/src/org/godotengine/godot/GodotFlurry.java:5: error: package com.flurry.android does not exist
import com.flurry.android.FlurryAgent;
                         ^
/home/kubeczek/git/godotBase/modules/flurry/android/src/org/godotengine/godot/GodotFlurry.java:19: error: cannot find symbol
            FlurryAgent.logEvent(p_name, null, p_timed);
            ^
  symbol:   variable FlurryAgent
  location: class GodotFlurry
/home/kubeczek/git/godotBase/modules/flurry/android/src/org/godotengine/godot/GodotFlurry.java:28: error: cannot find symbol
        FlurryAgent.logEvent(p_name, params, p_timed);
        ^
  symbol:   variable FlurryAgent
  location: class GodotFlurry
/home/kubeczek/git/godotBase/modules/flurry/android/src/org/godotengine/godot/GodotFlurry.java:49: error: cannot find symbol
        FlurryAgent.endTimedEvent(p_name);
        ^
  symbol:   variable FlurryAgent
  location: class GodotFlurry
/home/kubeczek/git/godotBase/modules/flurry/android/src/org/godotengine/godot/GodotFlurry.java:66: error: cannot find symbol
                FlurryAgent.onStartSession(activity, key);
                ^
  symbol: variable FlurryAgent
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
5 errors
:compileDebugJava FAILED

```

In the end I'm clueless was consulting this on irc with person (vaded) that have some experience with android modules and he was unable to find the reason as well.
I'm attaching my version of the module below:
[flurry.zip](https://github.com/punto-/godot_modules/files/282613/flurry.zip)

**Comment by kubecz3k** [comment]
Should be fixed by https://github.com/punto-/godot_modules/pull/3 the main problem was that it seems in Godot 2.0 the path to the jars should be relative to the `gradlew` (platform/android/java). 
So in this case it looks like this: `env.android_add_dependency("compile files('../../../modules/flurry/android//FlurryAgent.jar')")`

**Comment by kubecz3k** [comment]
fixed by #3
